src/scores/categorical/contingency_impl.py

```diff
@@ -150,11 +150,10 @@ def frequency_bias(self) -> xr.DataArray:
             \\text{frequency bias} = \\frac{\\text{true positives} + \\text{false positives}}{\\text{true positives} + \\text{false negatives}}
 
         Notes:
-
-            - Range: 0 to ∞ (infinity), where 1 indicates a perfect score
-            - "True positives" is the same as "hits"
-            - "False positives" is the same as "false alarms"
-            - "False negatives" is the same as "misses"
+            - Range: 0 to ∞ (infinity), where 1 indicates a perfect score.
```

Comment:
Thanks for the comment. I'm going to leave it in as an explanation in case someone doesn't recognise the symbol. It's pretty small on my screen, for example, and sometimes people's unicode might break. But I'm glad the symbol is doing a good job of rendering.